src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/custom.py

```diff
@@ -23,11 +23,24 @@
 
 
 # CUSTOM TYPE
-class KeyType(Enum):  # pylint: disable=too-few-public-methods
+class KeyType(Enum):
     primary = 'primary'
     secondary = 'secondary'
 
 
+# This is a work around to simplify the permission parameter for access policy creation, and also align with the other
+# command modules.
+# The original AccessRights enum is a combination of below four basic access rights.
+# In order to avoid asking for comma & space separated string from user, a space separated list is supported for
+# assigning multiple permissions.
+# The underlying IoT SDK should handle this. However it isn't right now. Remove this after it is fixed in IoT SDK.
+class SimpleAccessRights(Enum):
```

Comment:
This is good, but I would recommend you consider moving this model (eventually) into your SDK.